### PR TITLE
[hotfix][docs] Typo fix in doc pages

### PR DESCRIPTION
**Pull request by yangjf2019**
## What is the purpose of the change

* Remove the extra space separator. 

## Brief change log

* I have modified some files in the _docs/content/docs_ directory.

## Verifying this change

*I checked the changes.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changes to the `CustomResourceDescriptors`: (no)
  - Core observer or reconciler logic that is regularly executed: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/content/docs/custom-resource/reference.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ This page serves as a full reference for FlinkDeployment custom resource definit
 | Parameter | Type | Docs |
 | ----------| ---- | ---- |
 | job | org.apache.flink.kubernetes.operator.api.spec.JobSpec | Job specification for application deployments/session job. Null for session clusters. |
-| restartNonce | java.lang.Long | Nonce used to manually trigger restart for the cluster/session job. In order to trigger  restart, change the number to anything other than the current value. |
+| restartNonce | java.lang.Long | Nonce used to manually trigger restart for the cluster/session job. In order to trigger restart, change the number to anything other than the current value. |
```
Comment (gyfora) [inline docs/content/docs/custom-resource/reference.md:70]: The reference page is generated based on JavaDocs, you need to fix it there, you cannot edit this page directly

**Review by gyfora — CHANGES_REQUESTED** [review]

File: docs/content/docs/custom-resource/reference.md (hunk 1 above)
Comment (yangjf2019) [reply docs/content/docs/custom-resource/reference.md:70]: > The reference page is generated based on JavaDocs, you need to fix it there, you cannot edit this page directly

Okay, Thanks for your review ,I got it, and will update it.

Comment (yangjf2019) [reply docs/content/docs/custom-resource/reference.md]: @gyfora It is a space caused by a line break in the doc. Is there any solution for this? Thank you!

Comment (gyfora) [reply docs/content/docs/custom-resource/reference.md]: The solution in that case would be improving the doc generator to automatically remove double spaces.

Comment (yangjf2019) [reply docs/content/docs/custom-resource/reference.md]: Okay ,I got it.

Comment (yangjf2019) [reply docs/content/docs/custom-resource/reference.md]: @gyfora Please take a look again, thanks!

**Comment by gyfora** [comment]
can you please run the maven build locally before submitting any more fixes? This puts really unnecessary strain on the CI system.

**Comment by gyfora** [comment]
there should not be any more changed files after running 
```
mvn clean install -Pgenerate-docs
```

**Comment by yangjf2019** [comment]
@gyfora I ran the maven build locally before committing.

File: flink-kubernetes-operator-api/pom.xml [hunk 2]
```diff
@@ -276,6 +276,14 @@ under the License.
...
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
+                </configuration>
+            </plugin>
```
Comment (gyfora) [inline flink-kubernetes-operator-api/pom.xml:286]: I don't understand why we make this change here.

**Review by gyfora — CHANGES_REQUESTED** [review]

File: flink-kubernetes-operator-api/pom.xml (hunk 2 above)
Comment (yangjf2019) [reply flink-kubernetes-operator-api/pom.xml:286]: I added this code as prompted by the IDE, it may not be required and has been removed.

Comment (yangjf2019) [reply flink-kubernetes-operator-api/pom.xml]: Executing the mvn command locally verifies that it passes.

`mvn clean install -Pgenerate-docs`

image
image

Comment (gyfora) [reply flink-kubernetes-operator-api/pom.xml]: what if you do git status after running the build?
I understand that it passes but it should not leave any new changes

Comment (gyfora) [reply flink-kubernetes-operator-api/pom.xml]: The CI checks that after running the doc generation there are no new changes in git, this is the check that fails

Comment (yangjf2019) [reply flink-kubernetes-operator-api/pom.xml]: Thanks ,this is git status after running the build.
image

Comment (yangjf2019) [reply flink-kubernetes-operator-api/pom.xml]: Or, I turn off the PR and resubmit it again.

Comment (yangjf2019) [reply flink-kubernetes-operator-api/pom.xml]: > there should not be any more changed files after running
> 
> ```
> mvn clean install -Pgenerate-docs
> ```

This file has been changed.

`docs/content/docs/custom-resource/reference.md`

**Pull request closed without merging**